react/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { SubmissionError } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import { fetchUsers, sendToken } from '../../../redux/users/actions';
+
+import LoginForm from './components/LoginForm';
+
+class LoginFormContainer extends Component {
+  componentDidMount() {
+    const { getUsers, dispatchToken, history } = this.props;
+    if (window.localStorage.getItem('token')) {
+      dispatchToken(window.localStorage.getItem('token'));
+      history.push('/game');
+    }
+    getUsers();
+  }
+
+  submit = values => {
+    const { users, dispatchToken, history } = this.props;
+    users.forEach(user => {
```

Comment:
the api has a `/login` endpoint, where you send the email and password and it responds you if they are ok, check the readme here https://github.com/wolox-training/afe-react/tree/master/react/tic-tac-toe-api
getting all the users locally with all their passwords should never happen.

also if this requests fails, you are throwing an error that is not being handled anywhere, so the app will crash, you should handle the case with invalid credentials showing an error to the user.

and move this to a login action in redux, which should receive user and password and forward to /game when successful